### PR TITLE
Create event-checklist.md

### PR DESCRIPTION
**Pull request by chris-short**
**Which issue(s) this PR fixes**:

N/A

Adding shell event checklist before we forget things again.

### CODE REVIEW

File: communication/marketing-team/event-checklist.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Pre-Event (KubeCon) Checklist
```
Comment (mbbroberg) [inline communication/marketing-team/event-checklist.md:1]:
```suggestion
TODO - this document is intended to outline all Contributor Communications team activities to support a major event, like KubeCon. Please add to the list as we learn. See the [Marketing Team README](./README.md) for info on the team.

# Pre-Event (KubeCon) Checklist
```

**Comment by mbbroberg** [comment]
Good call @chris-short -- I'm suggesting we add that context to the beginning of the doc. Could you review, edit/accept, then it's good to go. 

/hold
/lgtm

**Comment by onlydole** [comment]
This looks good to me! Thank you, @chris-short 🎉

**Comment by mbbroberg** [comment]
/hold cancel 

Womp womp commands

**Pull request merged**